meta_review/handler.py

```diff
@@ -0,0 +1,440 @@
+                                         participant.rank - rank))
+                participant.trend = participant.rank - rank
+            else:
+                self.logger.debug('%s has no rank before, thus no trend'
+                                  % participant.login)
+
+            # update rank
+            if participant.rank:
+                self.logger.debug('update %s rank, before: %d, after: %d'
+                                  % (participant.login, participant.rank, rank))
+            else:
+                self.logger.debug('update %s rank, before: N/A, after: %d'
+                                  % (participant.login, rank))
+            participant.rank = rank
+
+            # save in memory
+            self.participants[participant.login] = participant
+
+    def __update_weight_factors(self):
+        """
+        Based on history data and the current iteration, recalculate weight
+        factors (to be used in the next iteration)
+
+        The higher score a person has, the more impacts he has, thus his
+        meta-reviews are more valuable.
+
+        For example, in a previous iteration, Alice got 2 marks, Bob got
+        0.8 marks and Charlie got 10 marks. The calculation demo would
+        be as follows:
+
+        >>> c = [2, 0.8, 10]
+        >>> max_score = float(max(c))
+        >>> result = [i / max_score for i in c]
+        >>> print(result)
+        [0.2, 0.08, 1.0]
+        >>> result_adjust = [i * 0.9 + 0.1 for i in result]  # adjust
+        >>> result_rounded = [round(i, 3) for i in result_adjust]
+        >>> print(result_rounded)
+        [0.28, 0.172, 1.0]
+
+        Anyone who gets negative marks from previous run will have weight
+        factor of 0.
+
+        To conclude, the weight factor is a float number ranging from 0 to 1.
+        """
+        max_score = 1.0
+        # find max score
+        for key, participant in self.participants.items():
+            if participant.score > max_score:
+                max_score = float(participant.score)
+
+        # calculate weight factors
+        for key, participant in self.participants.items():
+            if participant.score < 0:
+                participant.weight_factor = 0
+            else:
+                participant.weight_factor = participant.score / max_score
+                participant.weight_factor *= 0.9
+                participant.weight_factor += 0.1
+
+
+def handle():
+    # load data from gh-board repo
+    org_name = get_org_name()
+
+    # URL to grab all issues from
+    issues_url = 'http://' + org_name + '.github.io/gh-board/issues.json'
+
+    logger = logging.getLogger(__name__)
+
+    try:
+        content = requests.get(issues_url, timeout=10)
+        content.raise_for_status()
+    except Exception as ex:
+        logger.warning('Get issues from %s failed: %s Try backup url'
+                       % (issues_url, ex))
+        issues_url = 'https://' + org_name + '-gh-board.netlify.com/issues.json'
```

Comment:
If this fallback mechanism is moving to `community.git`, it should work correctly for the `community` repo, as that netlify is called `coala-community`, so the naming convention is the same.